apps/studio/components/interfaces/Integrations/Wrappers/Wrappers.constants.ts

```diff
@@ -3453,4 +3454,255 @@ export const WRAPPERS: WrapperMeta[] = [
       },
     ],
   },
+  {
+    name: 'hubspot_wrapper',
+    description:
+      'Query and sync HubSpot CRM data (companies, contacts, deals, tickets, and more) with Postgres using the Wasm FDW.',
```

Comment:
Thinking we can probably shorten this?

`Query and sync HubSpot CRM data using the Wasm FDW.`